test/selenium/lib/view/successful_page/accessors.js

```diff
@@ -12,7 +12,14 @@ function SuccessfulPageAccessor(driver) {
 
 SuccessfulPageAccessor.prototype = {
    get successMessageLocator() {
-        return this.driver.findElement(SELECTORS.successMessage);
+        var driver = this.driver;
```

Comment:
We currently [support 4.2](https://github.com/fxbox/foxbox/pull/222/files/fd5531433c1ccdf70e862dfcf602f584301b9cb5#diff-354f30a63fb0907d4ad57269548329e3R28)